doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -313,20 +333,18 @@ identᵢ’.
     Check tree_size_ind2.
 
 .. _derive-inversion:
-     
+
 Generation of inversion principles with ``Derive`` ``Inversion``
 -----------------------------------------------------------------
 
-The syntax of ``Derive`` ``Inversion`` follows the schema:
-
 .. cmd:: Derive Inversion @ident with forall (x : T), I t Sort sort
 
-This command generates an inversion principle for the `inversion … using` 
-tactic. Let `I` be an inductive predicate and `x` the variables occurring
-in t. This command generates and stocks the inversion lemma for the
-sort `sort` corresponding to the instance `∀ (x:T), I t` with the name
-`ident` in the global environment. When applied, it is equivalent to
-having inverted the instance with the tactic `inversion`.
+  This command generates an inversion principle for the `inversion … using`
```

Comment:
It would be best to write this ```:tacn:`inversion … using` ``` to have a direct link to the doc. This is not possible yet, because this tactic has no index entry. You can add one with the `:name:` option. (See the example of `induction ... using ...` in the same Tactics chapter.)